### ISSUE TITLE
Bootstrap script fails on Ubuntu 16.04

### ISSUE THREAD

**Issue by ocaisa** [easybuilders/easybuild-framework#1798]
The error is 

```
  File "/tmp/tmpa7s641/lib/python2.7/site-packages/distribute-0.6.49-py2.7.egg/pkg_resources.py", line 2430, in insert_on
    "with distribute. Found one at %s" % str(self.location))
ValueError: A 0.7-series setuptools cannot be installed with distribute. Found one at /usr/lib/python2.7/dist-packages
```

It looks like the same issue as [this one](https://github.com/pypa/setuptools/issues/543)

**Comment by boegel** [comment]
So, if I'm getting this right, if you have any `setuptools` version installed that has included `0.7` in it, you'll hit this problem... And it was fixed by simply bumping the version from 20.7.0 to 20.8.0 (and pretend it'll never happen again). Fantastic. >_<

@ocaisa which setuptools version is installed on this system? And can it be bumped?

I guess we could look into fixing this in the `distribute` that is used by the bootstrap script )which is one we patched already), but I'm not sure we want to go down that road...

**Comment by ocaisa** [comment]
Yup, you are right. 
I can bump it but only manually not from the distribution, we need Ubuntu to bump the version there.

**Comment by boegel** [comment]
@ocaisa Is this still relevant?

**Comment by ocaisa** [comment]
I think so, the system setuptools version in the Ubuntu 16.04 is still `20.7.0`

**Comment by boegel** [comment]
@ocaisa Yes, but the bootstrap script will now simply skip stage 0 is a sufficiently recent setuptools is already available on the system? See #1946

**Comment by branfosj** [comment]
The bootstrap script has been removed (see #4233), so I'll close this issue. See https://docs.easybuild.io/installation/ for the current instructions for installing EasyBuild.
